README.md

```diff
@@ -50,9 +58,12 @@ this method should set the pizza's temperature to be 300. Note: you may not use
 the variable pizza inside your method.
 
 Your Answer:
-```js
-// write code here
-```
+var pizza = {
+  temperature: 70,
+  bake: function () {
+    this.temperature = 300;
+    },
+};
```

Comment:
Since this document is formatted with markdown, please wrap code in three back tics (```). See: https://help.github.com/articles/markdown-basics/#code-formatting
```
var pizza = {
   ...
};
```